### DISCUSSION TITLE
How to make a spacer

### DISCUSSION THREAD

**Discussion by ffs-akl** [LBCrion/sfwbar#265] [Q&A]
I would like to make a spacer between the launcher and taskbar like the following photo.    how to implement it ?  a lable ? 
![swappy-20241123-193920](https://github.com/user-attachments/assets/9944dfbf-e413-407a-a0f5-e403a6360379)

**Comment by ffs-akl** [comment]
I can make a chart just like the cpu.widget,  is  there  lighter than chart?

**Comment by 01micko** [comment]
`label` is easier.

```
layout {
  label {
    style = "spacer"
    value = "\|" # escape the pipe char
  }
}

#CSS

label#spacer {
  /*min-width: 28px;
  min-height: 28px;*/
  color: @theme_selected_bg_color;
  font: bold 24px sans;
  padding: 6px;
}
```

Change to suit.

![spacer](https://github.com/user-attachments/assets/2414c9ef-3a31-44e9-ac19-0f1deb639ad0)

**Comment by LBCrion** [comment]
A blank label with a border on one side and -GtkWidget-vexpand: true

**Reply by 01micko** [reply]
I though about that one too.

**Reply by ffs-akl** [reply]
thank you both!   for me , all of two ways can solve my problem.  But GITHub only let me pick one as answer.  So I had to give up my options.

**Reply by 01micko** [reply]
Doesn't matter, as long as problem is solved :smile:
